package.json

```diff
@@ -9,7 +9,8 @@
     "@fluent/bundle": "^0.17.1",
     "@fluent/langneg": "^0.6.2",
     "@maxmind/geoip2-node": "^3.1.0",
-    "@sentry/node": "7.12.0",
+    "@sentry/node": "7.36.0",
+    "@sentry/tracing": "^7.36.0",
```

Comment:
should these two be installed in the `src/package.json` file instead (or in addition)?

Reply:
So the reason I'm putting sentry here is because V1 also needs sentry (and also should be upgraded). We can merge the two package.json when we launch v2